guvnor-inbox/guvnor-inbox-backend/src/test/java/org/guvnor/inbox/backend/server/security/InboxEntrySecurity_getInboxEntryRepositoryTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+package org.guvnor.inbox.backend.server.security;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.guvnor.common.services.project.service.ProjectService;
+import org.guvnor.inbox.backend.server.InboxEntry;
+import org.guvnor.structure.backend.repositories.ConfiguredRepositories;
+import org.guvnor.structure.organizationalunit.OrganizationalUnitService;
+import org.guvnor.structure.repositories.Repository;
+import org.jboss.errai.security.shared.api.identity.User;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.uberfire.backend.server.util.Paths;
+import org.uberfire.backend.vfs.Path;
+import org.uberfire.java.nio.file.FileSystem;
+import org.uberfire.java.nio.fs.file.SimpleFileSystemProvider;
+import org.uberfire.security.authz.AuthorizationManager;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+@RunWith( MockitoJUnitRunner.class )
+public class InboxEntrySecurity_getInboxEntryRepositoryTest {
```

Comment:
Please confirm this class name is intentional - I don't want @psiroky saying it looked funny again ;)